src/test/evo_deterministicmns_tests.cpp

```diff
@@ -344,6 +345,7 @@ BOOST_FIXTURE_TEST_CASE(dip3_protx, TestChain400Setup)
         BOOST_CHECK_EQUAL(chainTip->nHeight, nHeight + 1);
         SyncWithValidationInterfaceQueue();
         BOOST_CHECK(deterministicMNManager->GetListAtChainTip().HasMN(txid));
+        view = pcoinsTip.get();
```

Comment:
why so many view pointer updates? none of them should be needed.

Reply:
Yeah, not needed :+1: